va/va.h

```diff
@@ -2502,6 +2582,32 @@ VAStatus vaEndPicture (
     VAContextID context
 );
 
+/**
+ * Make the end of rendering for a pictures in contexts passed with submission. 
+ * The server should start processing all pending operations for contexts.
+ * All contexts passed should be associated through vaMFAddContext
+ * and call sequence Begin/Render/End performed.
+ * This call is non-blocking. The client can start another 
+ * Begin/Render/End/vaMFSubmit sequence on a different render targets.
```

Comment:
As current definition of VAEndPicture, if VAEndPicture is called, the server should start processing all pending operations. Seems it will conflict with vaMFSubmit, because before vaMFSubmit, VAEndPicture has processed current frame. The mean it requires change current implementation of VAEndPicture, or VAEndPicture should not be called for the case of multi-frame. Am I right?

Reply:
It doesn't require change to vaEndPicture, but to context working with multiframe. 
So context knows that it is working through multi-frame and doesn't perform submission during vaEndPicture

Reply:
"doesn't perform submission during vaEndPicture" seems conflict with the definition of VAEndPicture: " if VAEndPicture is called, the server should start processing all pending operations". Am I right? Any driver implementation should follow and make sure without any conflict with API definition.

Reply:
Yes, it conflicts with definition of vaEndPicture, but the same time all using vaMFSubmit will be aware of this change to behavior.

Reply:
so maybe there are two ways to avoid this conflict: 1. change the definition of VAEndPicture, add one more comment to indicate vaEndPicture won't process the pending operations in the MFP cases. 2. In the MFP case, no need to call vaEndPicture.

Reply:
Isn't it enough to have comment for vaMFSubmit?

Reply:
Posted example below why vaEndPicture behavior should be the way it described in vaMFSubmit